doc/computing/parallelism.rst

```diff
@@ -273,6 +273,26 @@ admissible seeds on your local machine:
 
     SKLEARN_TESTS_GLOBAL_RANDOM_SEED="all" pytest -v -k test_your_test_name
```

Comment:
This seems redundant/repeated now. Why don't we combine your addition with this section, and have it after the test example?